### DISCUSSION TITLE
Long Term Support (LTS)

### DISCUSSION THREAD

**Discussion by eyecatchup** [directus/directus#18709] [Feature Requests]
### Summary

An important criterion for IT decision makers for evaluating and adopting a software platform is support.

With regards to the recent license change, as an official Directus partner agency, we'd like to ask you to make your internal support policy (which I was told that exists since version 10) public / official, in the form of LTS releases. 

### Motivation

IT decision makers are responsible to discover the best software solution that works both now and in the long run. The availability of long-term support is one of a few important factors that affect that choice, because it means a better return on investment and a reduced risk. 

Long-term support not only extends the period of software maintenance, but also alters the type and frequency of software updates (patches) to reduce the risk, expense, and disruption of software deployment, while promoting the dependability of the software. Business customers are conservative when it comes to upgrades, they can calculate that a software supported long-term versus any unknown time is a much better value and safer invest. 

As an official Directus partner agency, our motivation is to build more enterprise Directus projects. By starting to charge (our) customers a commercial license fee, one important expectation is to get long-term supported software. So, having Directus LTS releases would be a really important selling factor in terms of commercial licenses.

### Detailed Design

not applicable

### Must Have:
- Offer LTS releases

### Drawbacks

not applicable

### Alternatives

not applicable

### Adoption Strategy

not applicable

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by metaforx** [reply]
I am currently evaluating Directus as an alternative to Django as it looks promising to develop web applications faster (no need for end-user administration interface, API as part of the system).

What holds me back is LTS, which is crucial if we want to use Directus for critical projects with a long lifespan (we are talking about 5years minimum). Providing detailed upgrade documentation is a big plus and saves many frustrating hours. I think effort by the core team in this field would be very beneficial for Directus eventually.

**Reply by shazib-summar** [reply]
Any updates here @rijkvanzanten ?

**Reply by rijkvanzanten** [reply]
Yup! Lemme close it out below

**Comment by rijkvanzanten** [comment]
We're starting with ESUs starting in this upcoming v10-v11 upgrade and will continue to do so for coming major version releases. 

This means v10 will continue to get security updates during the remainder of 2024. See https://directus.io/blog/extended-security-updates-directus-10-12 for more information 🙂
